package.json

```diff
@@ -38,7 +38,8 @@
         "ts-jest": "^25.2.0",
         "vue": "^2.5.16",
         "vue-jest": "^3.0.5",
-        "vue-router-back-button": "^1.1.1"
+        "vue-router-back-button": "^1.1.1",
+        "webpack-cli": "^3.3.12"
```

Comment:
whats the need for these added packages?

Reply:
It was a required update for the app to run on my system

Reply:
When i had issues with npm then, I deleted and node module and started again